predictors/process.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import pandas as pd
+
+
+def add_all_predictors(predictor_subset, columns):
+    """Add in all predictors in columns to predictor_subset."""
+    other_predictors = []
+    for predictor in columns:
+        if predictor not in predictor_subset and 'FCH4' not in predictor:
+           other_predictors.append(predictor)
+        elif predictor == 'FCH4':
+            continue
+        elif 'FCH4' in predictor:
+            print(f"Ignoring predictor {predictor}.")
+
+    predictor_subset += other_predictors
+    predictor_subset.remove('all')
+    
+    return predictor_subset
+
+
+def process_wind_direction_predictor(X):
+    X = X.copy()
+    X['WD'] = X['WD'] / 180 * np.pi
+    notna = ~X['WD'].isna()
```

Comment:
Why would there be NaNs here? Thought all NaNs were dealt in preprocessing.

Reply:
The NaNs in _CH4_ are dealt with in preprocessing. Imputation of the predictors happens during training   (specifically [here](https://github.com/stanfordmlgroup/methane-gapfill-ml/blob/predictors/models/base.py#L16-L22)) and testing once I implement that in the next PR!!